src/@batch-flask/utils/secure-utils.ts

```diff
@@ -31,6 +31,31 @@ export class SecureUtils {
         return retVal;
     }
 
+    public static passwordWindowsValid(length: number = 15): string {
```

Comment:
I would maybe have this in validators instead of secure-utils also shouldn't this return a `boolean`?

Reply:
This function is a generator for a new password that passes the validation requirements for Windows. I can rename the function to be more clear.

Reply:
oh I seem, yeah let's keep it here and have it named `generateWindowsPassword`